chatwoot/chatwoot
# feat: API to Filter reports by teams

## Description

This feature allows the user to filter reports by teams along with account labels, agents and inboxes.

Fixes #2916

## Type of change


## How Has This Been Tested?

1. Existing reports API's should function properly after the API changes.
2. Verify the team reports/summary API's via Postman.
3. Verify the CSV downloading API for teams.

## Checklist:
